lib/monitor.js

```diff
@@ -121,10 +121,15 @@ internals.Monitor.prototype.start = function (callback) {
         };
     };
 
+    var reportersIndex = 0;
+
     Items.serial(this.settings.reporters, function (item, next) {
 
         var reporter;
 
+        // Use hint of reporter index
+        var errorHint = self.settings.reporters.length > 1 ? '(' + reportersIndex++ + ')' : '';
```

Comment:
Couldn't you just start `reportersIndex` at 1 instead of having a conditional check in every iteration? Also since these errors are for developers, couldn't we just 0-based indexing like JS already uses?